src/com/palmergames/bukkit/towny/Towny.java

```diff
@@ -247,8 +247,6 @@ public void loadFoundation(boolean reload) {
 		loadLocalization(reload);
 		// Then load permissions
 		loadPermissions(reload);
-		// Unregister PAPIExpansion.
```

Comment:
All of the PAPI related calls in this class seem to have been removed but not replaced anywhere else. Have you broken PAPI support?